wye/profiles/urls.py

```diff
@@ -4,8 +4,8 @@
 urlpatterns = [
     url(r'^dashboard$', views.UserDashboard.as_view(),
         name="dashboard"),
-    url(r'^(?P<slug>[a-zA-Z0-9]+)/$',
+    url(r'^(?P<slug>[a-zA-Z0-9@.]+)/$',
         views.ProfileView.as_view(), name='profile-page'),
-    url(r'^(?P<slug>[a-zA-Z0-9]+)/edit$',
+    url(r'^(?P<slug>[a-zA-Z0-9@.]+)/edit$',
```

Comment:
@staranjeet @vnbang2003 Emails can also have `+` sign in them. 
Also I think using email in url is good idea. I would suggest using the user_id in the profile url if username is not present.